clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -211,7 +229,13 @@ const usePrivacyRequestForm = ({
       ),
       ...Object.fromEntries(
         Object.entries(identityInputs)
-          .filter(([key]) => key !== "email" && key !== "phone")
+          .filter(
+            ([key, value]) =>
+              key !== "email" &&
+              key !== "phone" &&
+              key !== "name" &&
+              typeof value !== "string"
+          )
```

Comment:
Filter out the default identities or identity inputs with just string values (such as "required" or "optional") from the dynamic form validation